terraform/charm/sharded/outputs.tf

```diff
@@ -0,0 +1,44 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+# Names of deployed applications
+output "app_names" {
+  description = "Names of of all deployed applications."
+  value = {
+    mongodb_config_server = module.mongodb_config_server.app_names["mongodb"]
+    shards                = [
+      for shard_module in module.mongodb_shards : shard_module.app_names["mongodb"]
+    ]
+  }
+}
+
+# Provided integration endpoints
+output "provides" {
+  description = "Map of all \"provides\" endpoints"
+  value = {
+    database      = "database"
+    config_server = "config-server"
+    cluster       = "cluster"
+    cos_agent     = "cos-agent"
+  }
+}
+
+# Required integration endpoints
+output "requires" {
+  description = "Map of all \"requires\" endpoints"
+  value = {
+    sharding                  = "sharding"
+    certificates              = "certificates"
+    s3_credentials            = "s3-credentials"
+    ldap                      = "ldap"
+    ldap_certificate_transfer = "ldap-certificate-transfer"
```

Comment:
This is not yet on 6/stable, so I don't know if we should add it right after release ? Or keep it already ?